app/views/devices_guidance/how_to_order_for_independent_special_schools.html.erb

```diff
@@ -0,0 +1,215 @@
+<% content_for :devices_service, true %>
+<% content_for :title, 'Order laptops and get internet access for state-funded pupils in independent special schools and alternative provision' %>
+
+<%- content_for :before_content do %>
+  <% breadcrumbs([{ "Home" => root_path },
+                  "Get laptops",
+                 ]) %>
+<% end %>
+
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Order laptops and get internet access for <span class="app-no-wrap">state-funded</span> pupils in independent special schools and alternative provision
+    </h1>
+
+    <p class="govuk-body">
+      Order laptops and get internet access for <span class="app-no-wrap">state-funded</span> pupils in independent special
+      schools and alternative provision.
+    </p>
+    <p class="govuk-body">
+      This guidance explains which pupils are eligible to receive laptops, and how local authorities (LAs)
+      can place orders through the Department for Education’s Get help with technology service.
+    </p>
+    <p class="govuk-body">
+      Independent special schools, alternative providers and pupils cannot order directly.
+    </p>
+    <p class="govuk-body">
+      We recommend that you get set up to order as soon as possible to make sure you get your preferred choice of laptops.
+    </p>
+
+    <h2 class="govuk-heading-l">
+      Which pupils are eligible for this support
+    </h2>
+    <p class="govuk-body">
+      The Department for Education (DfE) is providing laptops to support disadvantaged pupils in years 3 to 11, who:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>have a funded place at an independent special school, or independent alternative provision</li>
+      <li>are eligible for free school meals</li>
+      <li>do not have access to a suitable device for remote education, and for use in face-to-face education</li>
+    </ul>
+
+    <p class="govuk-body">
+      LAs can [request internet access for eligible pupils](#) too.
```

Comment:
Where should this link to?

Reply:
https://get-help-with-tech.education.gov.uk/internet-access